staging/src/k8s.io/legacy-cloud-providers/azure/auth/azure_auth.go

```diff
@@ -151,3 +169,18 @@ func decodePkcs12(pkcs []byte, password string) (*x509.Certificate, *rsa.Private
 
 	return certificate, rsaPrivateKey, nil
 }
+
+// azureStackOverrides ensures that the Environment matches what AKSe currently generates for Azure Stack
+func azureStackOverrides(env *azure.Environment, resourceManagerEndpoint, identitySystem string) {
+	env.ManagementPortalURL = strings.Replace(resourceManagerEndpoint, "https://management.", "https://portal.", -1)
+	// TODO: figure out why AKSe does this, why is autorest not setting ServiceManagementEndpoint?
+	env.ServiceManagementEndpoint = env.TokenAudience
+	// TODO: figure out why AKSe does this, may not be required, ResourceManagerVMDNSSuffix is not referenced anywhere
```

Comment:
why AKSe is related here? could we figure out whether the option is required or not?

Reply:
Tracked to this PR https://github.com/Azure/go-autorest/pull/93
Property was added to fix a `packer` issue.

No usage in cloud provider, go sdk or autorest.
Removing comment.